aplus/urls.py

```diff
@@ -31,7 +31,7 @@
 #  Pay attention to the order the URL patterns will be matched!
 urlpatterns = [
     url(r'^admin/', admin.site.urls),
-    url(r'^shibboleth/', include(shibboleth_login.urls)),
+    url(r'^', include(shibboleth_login.urls)),
```

Comment:
Is `^` required here when it is otherwise empty? 

By the way, `url` is deprecated and an alias to the newer `path_re` function. If regex is not needed, the more simple `path` function is preferred. As you can see, we have never refactored old urls to the new path functions.

Reply:
Yep, but I thought to keep it similar to several other entries on the same file. I also noticed about the deprecated API, but maybe we should to the refactoring as separate commit? (I can take that next on my todo list after this, if we want to do it for June release)

Reply:
Refactoring is not urgent. The current code is fine for this PR.